api/handlers.go

```diff
@@ -42,8 +54,14 @@ func CreateSessionHandlerFunc(sessions Sessions) http.HandlerFunc {
 		}
 
 		log.Event(ctx, "session created", log.Data{"session": sess}, log.INFO)
+		s := &session.Session{
+			ID:    sess.ID,
+			Email: sess.Email,
+			Start: sess.Start,
+		}
+		cache.Set(s)
```

Comment:
Add an error return to this method and then add the error checking/status codes etc. You will also need to update the unit tests to account for this scenario.